File: LeetCode/217. Contains Duplicate/217. Contains Duplicate.java
class Solution {
    public boolean containsDuplicate(int[] nums) {
        Map<Integer, Integer> countNum = new HashMap<Integer, Integer>();
        for(int i = 0; i < nums.length; i++) {
            if(countNum.containsKey(nums[i])) {
                return true;
            }
            countNum.put(nums[i], 1);
        }
        return false;
    }
}

File: LeetCode/217. Contains Duplicate/217. Contains Duplicate.cpp
class Solution {
    public:
        bool containsDuplicate (vector <int> &nums) {
            map <int, int> mp;
            for (int i = 0; i < nums.size(); i++) {
                if(!mp.count(nums[i])) {
                    mp[nums[i]] = 1;
                }
                else {
                    return true;
                }
            }
            return false;
        }
};